### PR TITLE
[AD-278] Request dialog

### PR DESCRIPTION
**Pull request by maksbotan**
**YT issue:** https://issues.serokell.io/issue/AD-278

**Checklist:**

- [x] Updated docs if necessary
  - [x] [README](README.md)
  - [x] [TUI usage guide](docs/usage-tui.md)
  - [x] [Configuration documentation](docs/configuration.md)
- [x] Adressed HLint warnings and hints
- [x] Rebased branch on current master and squashed all "Address PR comment" commits
- [x] Tested my changes if they modify code

**Description:**

### CODE REVIEW

File: ui/qt-app/Glue.hs [hunk 1]
```diff
@@ -89,6 +89,8 @@ knitFaceToUI UiFace{..} KnitFace{..} =
           return ()
       }
 
+    justOptNumber key = maybe [] (\value -> [Knit.ArgKw key . Knit.ExprLit . Knit.toLit . Knit.LitNumber $ fromIntegral value])
```
Comment (gromakovsky) [inline ui/qt-app/Glue.hs]: Could you please write type of this function? It's not obvious (I mean, it took me some time to figure it out).

File: ui/qt-lib/src/Ariadne/UI/Qt/Util.hs [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+import Ariadne.UI.Qt.Face
+
+formatBalance :: Text -> UiCurrency -> Text
+formatBalance balance unit = balance <> " " <> unitToHtml unit
+
+unitToHtml :: UiCurrency -> Text
+unitToHtml ADA = "<img src=':/images/ada-symbol-big-dark.png'>"
+unitToHtml Lovelace = "Lovelace"
+
```
Comment (gromakovsky) [inline ui/qt-lib/src/Ariadne/UI/Qt/Util.hs]: AFAIK this empty line shouldn't exist :man_shrugging:

Comment (maksbotan) [reply ui/qt-lib/src/Ariadne/UI/Qt/Util.hs]: Oops.

File: ui/qt-app/Glue.hs (hunk 1 above)
Comment (maksbotan) [reply ui/qt-app/Glue.hs]: It's just copy-paste from vty glue :man_shrugging: 
Will add.

**Pull request merged**